spec/javascript/orangelight/orangelight_ui_loader.spec.js

```diff
@@ -28,7 +30,26 @@ describe('OrangelightUiLoader', function() {
     const spy2 = jest.spyOn(FiggyManifestManager, 'buildMonogramThumbnails')
     let l = new loader
     l.setup_viewers()
-    expect(spy).not.toHaveBeenCalled()
-    expect(spy2).not.toHaveBeenCalled()
+    expect(spy).toHaveBeenCalledTimes(0)
+    expect(spy2).toHaveBeenCalledTimes(0)
```

Comment:
What does this change do?  I would have expected these to be the same as `not.toHaveBeenCalled`.

Reply:
@sandbergja thank you . It makes sure that it calls it once or twice etc. Otherwise toHaveBeenCalled is only checking if it is called. It's good to have an idea of how many times it's been called.

Reply:
👍